plugins/modules/inventory_intent.py

```diff
@@ -2433,6 +2425,71 @@ def get_provision_wired_device(self, device_ip):
 
         return True
 
+    def clear_mac_address(self, interface_id, deploy_mode, interface_name):
+        """
+        Clear the MAC address table on a specific interface of a device.
+        Parameters:
+            self (object): An instance of a class used for interacting with Cisco Catalyst Center.
+            interface_id (str): The UUID of the interface where the MAC addresses will be cleared.
+            deploy_mode (str): The deployment mode of the device.
+            interface_name(str): The name of the interface for which the MAC addresses will be cleared.
+        Returns:
+            self (object): An instance of a class used for interacting with Cisco Catalyst Center.
+        Description:
+            This function clears the MAC address table on a specific interface of a device.
+            The 'deploy_mode' parameter specifies the deployment mode of the device.
+            If the operation is successful, the function returns the response from the API call.
+            If an error occurs during the operation, the function logs the error details and updates the status accordingly.
+        """
+
+        try:
+            payload = {
+                "operation": "ClearMacAddress",
+                "payload": {}
+            }
+            clear_mac_address_payload = {
+                'payload': payload,
+                'interface_uuid': interface_id,
+                'deployment_mode': deploy_mode
+            }
+            response = self.dnac._exec(
+                family="devices",
+                function='clear_mac_address_table',
+                op_modifies=True,
+                params=clear_mac_address_payload,
+            )
+            self.log("Received API response from 'clear_mac_address_table': {0}".format(str(response)), "DEBUG")
+
+            if response and isinstance(response, dict):
+                task_id = response.get('response').get('taskId')
+
+                while True:
+                    execution_details = self.get_task_details(task_id)
+
+                    if execution_details.get("isError"):
+                        self.status = "failed"
+                        failure_reason = execution_details.get("failureReason")
+                        if failure_reason:
+                            self.msg = "Clearing the Mac address table for the interface '{0}' get failed because of {1}".format(interface_name, failure_reason)
+                        else:
+                            self.msg = "Clearing the Mac address table for the interface '{0}' get failed.".format(interface_name)
```

Comment:
self.msg = "Failed to clear the Mac address table for the interface '{0}'".format(interface_name)